tests/functional/compile/test_compile.py

```diff
@@ -39,6 +40,7 @@ def test_default(self, project):
         assert get_lines("first_model") == ["select 1 as fun"]
         assert any("_test_compile as schema" in line for line in get_lines("second_model"))
 
+    @pytest.mark.skip("Investigate flaky test #7179")
```

Comment:
Can we try to prioritize this issue before the 1.5 release, feels like something we would want to have coverage on.